packages/sdk/client/src/client.ts

```diff
@@ -75,39 +74,23 @@ export class Client {
    * Unique id of the Client, local to the current peer.
    */
   private readonly _instanceId = PublicKey.random().toHex();
-
-  // prettier-ignore
-  constructor({
-    config,
-    modelFactory,
-    services
-  }: ClientOptions = {}) {
+  constructor(options: ClientOptions = {}) {
     if (
       typeof window !== 'undefined' &&
       window.location.protocol !== 'https:' &&
       !window.location.origin.includes('localhost')
```

Comment:
window.locaiton.host === 'localhost'

Reply:
I think there was a specific reason for this...I think it was the kube localhost domains. Maybe this should be `endsWith`?